this test isnt even run automatically and the dependencies are all broken,
so just disable it and be done

--- zziplib-0.13.69/SDL/Makefile.am
+++ zziplib-0.13.69/SDL/Makefile.am
@@ -5,7 +5,7 @@
 zzipsdl_DATA = SDL_rwops_zzip.c SDL_rwops_zzip.h
 
 EXTRA_DIST = $(sdl_DATA)
-check_PROGRAMS = zzcatsdl
+check_PROGRAMS = 
 
 zzcatsdl_SOURCES = SDL_rwops_zzcat.c SDL_rwops_zzip.c
 ZZIP_CFLAGS= `export PKG_CONFIG_PATH=../zzip; pkg-config zziplib --cflags` 
